acl_loader/main.py

```diff
@@ -409,10 +417,17 @@ def convert_l2(self, table_name, rule_idx, rule):
             else:
                 try:
                     rule_props["ETHER_TYPE"] = int(rule.l2.config.ethertype)
-                except:
-                    raise AclLoaderException("Failed to convert ethertype %s table %s rule %s" % (
+                except Exception:
+                    raise AclLoaderException("Failed to convert ethertype %s; table %s rule %s" % (
                         rule.l2.config.ethertype, table_name, rule_idx))
 
+        if rule.l2.config.vlan_id:
+            try:
+                rule_props["VLAN_ID"] = int(rule.l2.config.vlan_id)
+            except Exception:
```

Comment:
>Exception [](start = 19, length = 9)

It ignores the original exception, we need at least log it or print to stderr.

Reply:
Done